code/modules/reagents/chemistry/reagents/blob_reagents.dm

```diff
@@ -179,54 +162,34 @@
 	if(iscarbon(M))
 		M.emote("scream")
 
-/datum/reagent/blob/boiling_oil/extinguish_reaction(obj/effect/blob/B)
+/datum/reagent/blob/blazing_oil/extinguish_reaction(obj/effect/blob/B)
 	B.take_damage(rand(1, 3), BURN)
 
-//does burn and toxin damage, explodes into flame when hit with burn damage
-/datum/reagent/blob/flammable_goo
-	name = "Flammable Goo"
-	id = "flammable_goo"
-	description = "will do medium burn and toxin damage, make targets flammable, and ignite when burned."
-	shortdesc = "will do medium burn and toxin damage, and make targets flammable."
-	analyzerdescdamage = "Does medium burn damage, medium toxin damage, and makes targets flammable."
-	analyzerdesceffect = "Releases bursts of flame when attacked with burn damage, but takes additional burn damage."
-	color = "#BE5532"
-	complementary_color = "#329BBE"
-	blobbernaut_message = "splashes"
-	message = "The blob splashes you with a thin goo"
-	message_living = ", and you smell a faint, sweet scent"
-
-/datum/reagent/blob/flammable_goo/reaction_mob(mob/living/M, method=TOUCH, reac_volume, show_message, touch_protection, mob/camera/blob/O)
-	reac_volume = ..()
-	M.adjust_fire_stacks(round(reac_volume/8)) //apply, but don't ignite
-	M.apply_damage(0.4*reac_volume, TOX)
-	if(M)
-		M.apply_damage(0.4*reac_volume, BURN)
-
-/datum/reagent/blob/flammable_goo/damage_reaction(obj/effect/blob/B, original_health, damage, damage_type, cause)
+/datum/reagent/blob/blazing_oil/damage_reaction(obj/effect/blob/B, original_health, damage, damage_type, cause)
 	if(cause && damage_type == BURN)
 		for(var/turf/open/T in range(1, B))
 			var/obj/effect/blob/C = locate() in T
 			if(!(C && C.overmind && C.overmind.blob_reagent_datum.id == B.overmind.blob_reagent_datum.id) && prob(80))
 				PoolOrNew(/obj/effect/hotspot, T)
-		return damage * 1.5
 	return ..()
 
-//does toxin damage, targets think they're not hurt at all
+//does toxin damage, hallucination, targets think they're not hurt at all
 /datum/reagent/blob/regenerative_materia
 	name = "Regenerative Materia"
 	id = "regenerative_materia"
-	description = "will do low toxin damage and cause targets to believe they are fully healed."
-	analyzerdescdamage = "Does low toxin damage and injects a toxin that causes the target to believe they are fully healed."
+	description = "will do low toxin damage, hallucinations, and cause targets to believe they are fully healed."
+	analyzerdescdamage = "Does low toxin damage and injects a toxin that causes the target to hallucinate they are fully healed."
```

Comment:
Bad idea to mix hallucinations and the full heal thing.  The hallucination random health will override the fake full health.